modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/views/ChangeTrackingRenderView.js

```diff
@@ -1337,7 +1337,7 @@ export default function ChangeTrackingRenderView({
 			);
 		}
 
-		if (workflowStatus && showWorkflow) {
+		if (workflowStatus !== null && showWorkflow) {
```

Comment:
If parenthesis is allowed, let's add it around `workflowStatus !== null` so it's easier to read.

Reply:
Ok, let's leave it as is then.